cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -294,6 +304,15 @@ public void execute() {
         }
     }
 
+    private boolean isProjectEmpty(Project project) {
+        for (ModuleId moduleId : project.currentPackage().moduleIds()) {
+            if (!project.currentPackage().module(moduleId).documentIds().isEmpty()) {
```

Comment:
We should continue to build if the module has only test documents. That part is missing here. Return `true` only if the module neither has source documents nor any test documents. You can use `module.testDocumentIds()` for this

Reply:
Added `module.testDocuementIds()` to the filter